include/mme-app/ue_table.h

```diff
@@ -85,6 +86,7 @@ enum ue_proc{
 };
 
 enum ecm_states{
+ ECM_UNKNOWN=0,
```

Comment:
Add a comment which says when we move to IDLE state and when we move to connected state.

Reply:
done.